doc/swagger.yaml

```diff
@@ -2198,12 +2198,12 @@ info:
     email: tbd-developer@squareup.com
     name: TBD
     url: https://github.com/TBD54566975/ssi-service/issues
-  description: '{{.Desc}}'
+  description: 'The Self Sovereign Identity Service: Managing DIDs, Verifiable Credentials,
+    and more!'
   license:
     name: Apache 2.0
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   title: SSI Service API
-  version: '{{.SVN}}'
```

Comment:
I think we should PIN this in source code, and remove from the configuration file. What's the purposed of having it be configurable? We have no logic that's making decisions off of it.

Reply:
agreed

Reply:
we have this issue to track https://github.com/TBD54566975/ssi-service/issues/502 don't want to overload this change set